src/Form.js

```diff
@@ -0,0 +1,52 @@
+import React, {memo, useState} from 'react';
+import styled from "styled-components";
+
+export const StyledForm = styled.form`
+    display: flex;
+    width: 270px;
+    white-space: nowrap;
+    align-items: center;
+    align-self: flex-end;
+`
+export const Input = styled.input`
+    width: 70vw;
+    margin: 10px;
+    margin-left: 0px;
+    height: 30px;
+    border: solid 1px #dcdcdc;
+    border-radius: 10%;
+`
+export const Button = styled.button`
+    width: 10vw;
+    height: 35px;
+    background: yellow;
+    border: 0;
+    border-radius: 20px;
+    margin-right: 5px;
+`
+const Form = memo(({ isMe, upLoadChatting, chatting }) => {
+    const [text, setText] = useState('');
+
+    const onChangeText = (e) => {
+        setText(e.target.value)
+    };
+
+    const onSubmitHandler = (e) => {
+        e.preventDefault();
+        if (text==='') {
+            alert('type something!!');
+            return;
+        }
+        upLoadChatting([...chatting, {isMe, text}]);
+        setText('');
+    };
+
+    return(
+        <StyledForm onSubmit={onSubmitHandler}>
+            <Input placeholder="Message..." value={text} onChange={onChangeText} />
+            <Button>전송</Button>
+        </StyledForm>
```

Comment:
```suggestion
        <Wrapper onSubmit={onSubmitHandler}>
            <Input placeholder="Message..." value={text} onChange={onChangeText} />
            <Button>전송</Button>
        </Wrapper>
```

한 파일 안에 하나의 컴포넌트가 있다면, 그 컴포넌트의 styled-component는 Wrapper로 네이밍하는 경우가 많더라구요~!